### PR TITLE
Add some example of integrations

### PR DESCRIPTION
**Pull request by mathieupotier**
Update docker-run.sh to deal with docker secret for some vars.

### CODE REVIEW

**Comment by tuxgasy** [comment]
I think this PR adds a lot of complexity.

**Comment by mathieupotier** [comment]
Once this PR will be merged : https://github.com/tuxgasy/docker-dolibarr/pull/46
I'll update this one to keep only the docs, it's true that I mixed with the secrets integration, I should have done 2 separated PRs.

File: update.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-DOCKER_BUILD=${DOCKER_BUILD:-0}
+DOCKER_BUILD=${DOCKER_BUILD:-1}
```
Comment (tuxgasy) [inline update.sh:5]: do not build by default

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
+.idea/
```
Comment (tuxgasy) [inline .gitignore:1]: Why this?

**Review by tuxgasy — COMMENTED** [review]
Sorry for the delay.

I think it is not necessary commit all modifications in `images` directory for the next PR.
I will update it after merge.

File: .gitignore (hunk 2 above)
Comment (mathieupotier) [reply .gitignore:1]: It's my IDE local settings in there, I can use a global gitignore if you don't want this here ... just tell me

File: update.sh (hunk 1 above)
Comment (mathieupotier) [reply update.sh]: done

File: .gitignore (hunk 2 above)
Comment (tuxgasy) [reply .gitignore:1]: I prefer that you delete it. Thank you.

Comment (mathieupotier) [reply .gitignore]: done

**Review by tuxgasy — APPROVED** [review]

**Pull request merged**